spec/resources/cdl/charge_manager_spec.rb

```diff
@@ -21,6 +21,32 @@ def item_ids(source_metadata_identifier:)
 
   let(:change_set_persister) { ScannedResourcesController.change_set_persister }
 
+  describe "#initialize" do
+    def charged_item_count
+      Valkyrie.config.metadata_adapter.query_service.find_all_of_model(model: CDL::ChargedItem).count
+    end
+
+    it "cleans up expired charges" do
+      stub_bibdata(bib_id: "123456")
+      resource = FactoryBot.create_for_repository(:scanned_resource, source_metadata_identifier: "123456")
+      eligible_item_service = EligibleItemService.new(item_ids: ["1234"])
+
+      Timecop.freeze(Time.current - 1.hour) do
+        charged_items = [
+          FactoryBot.create_for_repository(:charged_item, item_id: "1234", netid: "zelda", expiration_time: Time.current + 1.hour)
```

Comment:
You can just do `FactoryBot.build`, since these are just embedded in the resource.